### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by lchastanet**
Finals features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,57 @@
...
+
+Then load the initial dataset into database
+
+```
+php bin/console doctrine:fixtures:load
+```
+
+And finally run the dev server to test app
+
+- if you have installed the symfony cli tool :
```
Comment (friexo) [inline README.md:48]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/friexo/p7-daphp-oc/pullRequest?prid=6399926)

**Comment by friexo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- src/Controller/ClientController.php  4
- src/Controller/ProductController.php  4
- src/Controller/UserController.php  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/friexo/p7-daphp-oc/pullRequest?prid=6399926&bid=20749016)

File: README.md [hunk 2]
```diff
@@ -2,3 +2,57 @@
...
+
+Then run the following command to create database
+
+```shell
+php bin/console doctrine:database:create
+```
+
+Then load the initial dataset into database
+
+```
```
Comment (friexo) [inline README.md:42]: Codacy found an issue: [[fenced-code-flag] Missing code language flag](https://app.codacy.com/gh/friexo/p7-daphp-oc/pullRequest?prid=6399926)

File: README.md [hunk 3]
```diff
@@ -2,3 +2,57 @@
...
+
+```shell
+$ mkdir -p config/jwt
+$ openssl genpkey -out config/jwt/private.pem -aes256 -algorithm rsa -pkeyopt rsa_keygen_bits:4096
+$ openssl pkey -in config/jwt/private.pem -out config/jwt/public.pem -pubout
+```
+
+
+
+Then create or modify the **.env** file to connect to your mysql installation and setup the mail server
```
Comment (friexo) [inline README.md:25]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 2 lines before node](https://app.codacy.com/gh/friexo/p7-daphp-oc/pullRequest?prid=6399926)

File: README.md [hunk 4]
```diff
@@ -2,3 +2,57 @@
...
+
+And finally run the dev server to test app
+
+- if you have installed the symfony cli tool :
+
+```shell
+symfony serve
+```
+
+- if you don't have installed the symfony cli tool :
```
Comment (friexo) [inline README.md:54]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/friexo/p7-daphp-oc/pullRequest?prid=6399926)

**Comment by lchastanet** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Controller/UserController.php  6
- src/Controller/ProductController.php  4
- src/Controller/ClientController.php  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/friexo/p7-daphp-oc/pullRequest?prid=6399926&bid=20749016)

**Pull request merged**